pkg/cmd/cli/cmd/volume.go

```diff
@@ -28,50 +31,52 @@ const (
 Update volumes on a pod template
 
 This command can add, update or remove volumes from containers for any object
-that has a pod template (replication controllers or deployment configurations).
+that has a pod template (deployment configs, replication controllers, or pods).
 You can list volumes in pod or any object that has a pod template.
 You can specify a single object or multiple, and alter volumes on all containers or
-just those that match a wildcard.`
+just those that match a given name.
 
-	volumeExample = `  // Add new volume of type 'emptyDir' for deployment config 'registry' and mount under /opt inside the containers
-  // The volume name is auto generated
-  $ %[1]s volume dc/registry --add --mount-path=/opt
-
-  // Add new volume 'v1' with secret 'magic' for replication controller 'r1'
-  $ %[1]s volume rc/r1 --add --name=v1 -m /etc --type=secret --secret-name=magic
+Volume types include:
 
-  // Add new volume to replication controller 'r1' based on git repository
-  // or other volume sources not supported by --type
-  $ %[1]s volume rc/r1 --add -m /repo --source=<json-string>
+* emptydir (empty directory)
+    A directory allocated when the pod is created on a local host, is removed when
+    the pod is deleted and is not copied across servers
+* hostdir (host directory)
+    A directory with specific path on any host (requires elevated privileges)
+* persistentvolumeclaim or pvc (persistent volume claim)
+    Link the volume directory in the container to a persistent volume claim you have
+    allocated by name - a persistent volume claim is a request to allocate storage.
+    Note that if your claim hasn't been bound, your pods will not start.
+* secret (mounted secret)
+    Secret volumes mount a named secret to the provided directory.
 
-  // Add emptyDir volume 'v1' to a deployment config definition on disk and 
-  // update the deployment config on the server
-  $ %[1]s volume -f dc.json --add --name=v1
+For descriptions on other volume types, see https://docs.openshift.com`
 
-  // Create a new persistent volume and overwrite existing volume 'v1' for replication controller 'r1'
-  $ %[1]s volume rc/r1 --add --name=v1 -t persistentVolumeClaim --claim-name=pvc1 --overwrite
+	volumeExample = `  // List volumes defined on all deployment configs in the current project
+  $ %[1]s volume dc --all
```

Comment:
Not strong about it but most examples in other commands use the full name of resources.